clients/ts/FunctionalTests/ts/WebSocketTests.ts

```diff
@@ -1,33 +1,37 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+import { WebSocketWrapper } from "@aspnet/signalr/dist/esm/Polyfills";
 import { ECHOENDPOINT_URL } from "./Common";
 
 // On slower CI machines, these tests sometimes take longer than 5s
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 10 * 1000;
 
-if (typeof WebSocket !== "undefined") {
-    describe("WebSockets", () => {
-        it("can be used to connect to SignalR", (done) => {
-            const message = "message";
+describe("WebSockets", () => {
+    it("can be used to connect to SignalR", (done) => {
+        const message = "message";
 
-            const webSocket = new WebSocket(ECHOENDPOINT_URL.replace(/^http/, "ws"));
+        let webSocket: WebSocket;
+        if (typeof WebSocket !== "undefined") {
+            webSocket = new WebSocket(ECHOENDPOINT_URL.replace(/^http/, "ws"));
+        } else {
+            webSocket = new WebSocketWrapper(ECHOENDPOINT_URL.replace(/^http/, "ws"));
```

Comment:
Use `require` to import the module **inside** the `if` condition that guarantees we're in node.